src/domain/myProfile/myProfileForm/ValidationSchema.ts

```diff
@@ -5,6 +5,9 @@ import { VALIDATION_MESSAGE_KEYS } from '../../app/i18n/constants';
 export const getMyProfileValidationSchema = (type: 'create' | 'edit') => {
   const commonFormSchema = {
     name: Yup.string().required(VALIDATION_MESSAGE_KEYS.TIME_REQUIRED),
+    emailAddress: Yup.string()
+      .required(VALIDATION_MESSAGE_KEYS.TIME_REQUIRED)
```

Comment:
Seems like `VALIDATION_MESSAGE_KEYS.TIME_REQUIRED` is used throughout this file. Maybe should be changed to avoid confusion :)